<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shipyard</class>
 <widget class="QMainWindow" name="Shipyard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shipyard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="statBox">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="nameEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Points</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="pointsSpin">
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="ucmCheck">
              <property name="text">
               <string>UCM</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="scourgeCheck">
              <property name="text">
               <string>Scourge</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="phrCheck">
              <property name="text">
               <string>PHR</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="shaltariCheck">
              <property name="text">
               <string>Shaltari</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="resistanceCheck">
              <property name="text">
               <string>Resistance</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Admiral Discount</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="admiralDiscountSpin">
              <property name="maximum">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="attrBox">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Scan</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="scanSpin"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Sig</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sigSpin"/>
            </item>
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>altSig</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="altSig_spin"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Thrust</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="thrustSpin"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Hull</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="hullSpin"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Armor</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="armorCombo"/>
            </item>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Passive</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="passiveCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>PD</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="pdSpin"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>GL</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="glSpin"/>
            </item>
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>GH</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="ghSpin"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Tonnage</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="tonnageCombo"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="periBox">
          <item>
           <layout class="QVBoxLayout" name="weaponBox">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_15">
                <item>
                 <widget class="QPushButton" name="newWeaponButton">
                  <property name="text">
                   <string>New</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="editWeaponButton">
                  <property name="text">
                   <string>Edit</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="copyWeaponButton">
                  <property name="text">
                   <string>Copy</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_16">
                <item>
                 <widget class="QPushButton" name="deleteWeaponButton">
                  <property name="text">
                   <string>Delete</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="clearWeaponsButton">
                  <property name="text">
                   <string>Clear</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="weaponRosterLayout"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="specialBox">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QPushButton" name="addSpecialButton">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deleteSpecialCombo">
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="specialNumSpin"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QComboBox" name="specialSelectCombo"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="specialLayout"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="launchBox">
        <item>
         <widget class="QPushButton" name="loadLaunchAssetsButton">
          <property name="text">
           <string>Load Assets</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="launchAssetLayout"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QPushButton" name="launchAddButton">
            <property name="text">
             <string>Add Selected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="launchAddSCButton">
            <property name="text">
             <string>Add Fighter Bombers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="launchDeleteButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Limited</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="limitedSpin"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Count</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="launchQuantitySpin"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="launchProfileLayout"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="optionBox">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="newOptionButton">
              <property name="text">
               <string>New</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editOptionButton">
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteOptionButton">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Broadside</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="maxBroadsideSpin"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Min</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="minOptionsSpin"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Max</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="maxOptionsSpin"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="optionLayout"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="saveShipButton">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteShipButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="loadShipButton">
          <property name="text">
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="shipRosterLayout"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1295</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
